api/src/main/java/org/openmrs/api/impl/OrderServiceImpl.java

```diff
@@ -1061,6 +1072,47 @@ public OrderType getOrderTypeByConceptClass(ConceptClass conceptClass) {
 	public OrderType getOrderTypeByConcept(Concept concept) {
 		return Context.getOrderService().getOrderTypeByConceptClass(concept.getConceptClass());
 	}
+
+	/**
+	 * @see org.openmrs.api.OrderService#getOrderTypesByClassName(String)
+	 */
+	@Override
+	@Transactional(readOnly = true)
+	public List<OrderType> getOrderTypesByClassName(String javaClassName) throws APIException {
+		return dao.getOrderTypesByClassName(javaClassName);
+	}
+	
+	/**
+	 * @see org.openmrs.api.OrderService#getOrderTypesByClassName(String, boolean, boolean)
+	 */
+	@Override
+	@Transactional(readOnly = true)
+	public List<OrderType> getOrderTypesByClassName(String javaClassName, boolean includeSubclasses, boolean includeRetired) throws APIException {
+		if (!includeSubclasses) {
+			return getOrderTypesByClassName(javaClassName);
```

Comment:
This doesn't use the "includeRetired" parameter, which would be incorrect in this case, right?  Maybe we need to include the includeRetired parameter in the getOrderTypesByClassName method too?